mygame/app/main.rb

```diff
@@ -7,7 +7,8 @@
 require 'app/scenes/battle.rb'
 require 'app/scenes/debug_screen.rb'
 
-LOWREZ_ZOOM = 11
+
+LOWREZ_ZOOM = 10
```

Comment:
Was there any problem with 11 pixels ? :D I personally thought the bigger the better...

Reply:
I'm obsessed with multiples of 10 and you just missed the opportunity of using the very first of them. How could you? HOW-COULD-YOU????
(:rofl: no, this is a leftover I missed of some play-around I was doing before we started :P)